### DISCUSSION TITLE
weights and normalized weights

### DISCUSSION THREAD

**Discussion by gsaha009** [columnflow/columnflow#483] [General]
Hello Experts (@riga, @nprouvost, @haddadanas : from `hh2bbtautau` team !),

I was looking at the `hh2bbtautau` columnflow based analysis  and I noticed two version s of the same wights, e.g. `pdf_weights`.
In `SelectEvents` the [`pdf_weights`](https://github.com/uhh-cms/hh2bbtautau/blob/f51e28dc442cd4edcb9471d0d12fbf790b79ca5e/hbt/selection/default.py#L135) producer from `columnflow.production.cms.pdf` is called --> it saves the weights --> nomalizes it in [`production.weights`](https://github.com/uhh-cms/hh2bbtautau/blob/master/hbt/production/weights.py#L116).

Can anyone please explain the reason behind using the normalized weights? And, what are the weights one has to use this normalization for?

Thank you :)

**Comment by mafrahm** [comment]
Hi @gsaha009, the idea of the renormaization is to separate the rate and shape effects of the PDF uncertainty and pass this as two separate nuisance parameters in the combine fit. The `pdf_weights` column corresponds to the shape-based uncertainty, but applying e.g. `pdf_up` can have an effect on the rate. This rate effect should be removed before applying any cuts, which is achieved by scaling the pdf weights to the "nominal" sum of weights.

**Reply by gsaha009** [reply]
Hi @mafrahm 
Thanks a lot :)
